### PR TITLE
[Orc8r] Add deploy tests to circleci config

### PR DESCRIPTION
**Pull request by karthiksubraveti**
Signed-off-by: Karthik Subraveti <ksubraveti@fb.com>

## Summary
Run deployment tests on every checkin. Also identified the chart version mismatch through 
these tests and fixed them.

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: .circleci/config.yml
Comment (hcgatewood) [inline .circleci/config.yml]: Turning this on should be fine

File: .circleci/config.yml
Comment (hcgatewood) [inline .circleci/config.yml]: I think this is unnecessary, both here and the other 1-2 places where it seems to be used for no purpose

**Review by hcgatewood — APPROVED** [review]
Lgtm, 2 comments before merging

**Review by hcgatewood — APPROVED** [review]
@karthiksubraveti ready to merge?

**Comment by karthiksubraveti** [comment]
> @karthiksubraveti ready to merge?

Need @tmdzk approval to merge

**Comment by 119Vik** [comment]
@karthiksubraveti  can you please check line length warning
I thought we are trying to migrate all jobs from CircleCi to GitHub actions so does it make sense for You to add this job to GitHub actions?

File: .circleci/config.yml
Comment (tmdzk) [inline .circleci/config.yml:579]: Why would you remove that?

File: .circleci/config.yml
Comment (tmdzk) [inline .circleci/config.yml:594]: Uncomment

File: .circleci/config.yml
Comment (tmdzk) [inline .circleci/config.yml:593]: Let's do Ubuntu 20

**Review by tmdzk — CHANGES_REQUESTED** [review]

File: .circleci/config.yml
Comment (tmdzk) [reply .circleci/config.yml:579]: Ok nvm just checked the job it is useless

File: .circleci/config.yml
Comment (karthiksubraveti) [reply .circleci/config.yml:594]: There was an issue which I uncovered with the tests failing when I uncommented that. I will delete this line to avoid confusion here.

Comment (tmdzk) [reply .circleci/config.yml:594]: Just remove it then

**Review by tmdzk — APPROVED** [review]

**Pull request merged**
